admin/class-metabox.php

```diff
@@ -33,19 +30,18 @@ public function __construct() {
 	 * the main meta box definition array in the class WPSEO_Meta() as well!!!!
 	 */
 	public static function translate_meta_boxes() {
-		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet Preview', 'wordpress-seo' );
+		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet', 'wordpress-seo' );
```

Comment:
@terw-dan please be complete in your explanation. Why does it have to be called "snippet"?